src/pylibsshext/includes/sftp.pxd

```diff
@@ -30,6 +32,15 @@ cdef extern from "libssh/sftp.h" nogil:
         pass
     ctypedef sftp_file_struct * sftp_file
 
+    struct sftp_attributes_struct:
+        char *name
+        char *longname
+        stdint.uint32_t flags
+        stdint.uint8_t type
+        stdint.uint64_t size
+        # ...
```

Comment:
```suggestion
```

Reply:
I wanted to express that this is not the whole structure as there are more members I did not include as we do not use them. Not sure if this is good practice though and rather than not including even this comment, I would include also the other members.

Reply:
Filled the whole structure instead